### PR TITLE
[openshift_ovn] Collect db files for ovn interconnect environment

### PR DESCRIPTION
**Pull request by pperiyasamy**
This updates openshift_ovn plugin to collect ovn db files and logs when it is running with interconnect configuration.

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/report/plugins/openshift_ovn.py [hunk 1]
```diff
@@ -27,6 +27,12 @@ def setup(self):
             "/var/log/openvswitch/libreswan.log",
             "/var/log/openvswitch/ovs-monitor-ipsec.log"
         ])
+        # Collect ovn interconnect specific files if exists.
+        self.add_copy_spec([
+            "/var/lib/ovn-ic/etc/ovnnb_db.db",
+            "/var/lib/ovn-ic/etc/ovnsb_db.db",
+            "/var/log/ovn-ic/acl-audit-log.log"
```
Comment (npinaeva) [inline sos/report/plugins/openshift_ovn.py:34]: why did you add this file? looks like we didn't collect it before

File: sos/report/plugins/openshift_ovn.py [hunk 2]
```diff
@@ -27,6 +27,12 @@ def setup(self):
...
+        ])
```
Comment (npinaeva) [inline sos/report/plugins/openshift_ovn.py:35]: I think `/var/run/ovn` folder also changed

Comment (pperiyasamy) [reply sos/report/plugins/openshift_ovn.py:34]: yes, but it is still mounted as /var/run/ovn/ into ovnkube-node container, so i guess we're fine there.

File: sos/report/plugins/openshift_ovn.py (hunk 1 above)
Comment (pperiyasamy) [reply sos/report/plugins/openshift_ovn.py]: oh! yes, removed it.

**Review by arif-ali — APPROVED** [review]

File: sos/report/plugins/openshift_ovn.py (hunk 2 above)
Comment (npinaeva) [reply sos/report/plugins/openshift_ovn.py:34]: the problem is sos report runs on the host, not in a container. You can test these commands e.g. if you `chroot /host` from the debug pod

Comment (npinaeva) [reply sos/report/plugins/openshift_ovn.py:34]: I guess we need to add collection for ovnkube-controller container?

Comment (pperiyasamy) [reply sos/report/plugins/openshift_ovn.py:34]: Yes @npinaeva , the container commands don't work anymore due to directory path mismatch. now collecting ovs `ct-zone-list` directly on the host which is working fine.

**Review by TurboTurtle — APPROVED** [review]
